src/proxy.c

```diff
@@ -1290,12 +1290,13 @@ cc_proxy_run_hyper_new_container (struct cc_oci_config *config,
 				  const char *container_id,
 				  const char *rootfs, const char *image)
 {
-	JsonObject *newcontainer_payload= NULL;
-	JsonObject *process = NULL;
-	JsonArray *args= NULL;
-	JsonArray *envs= NULL;
-	gchar     *uid_str = NULL;
-	gchar     *gid_str = NULL;
+	JsonObject *newcontainer_payload = NULL;
```

Comment:
You seem to have introduced some whitespace changes here too. It would be helpful if you could make such changes on a separate commit as again, this artificially make the diff bigger, and thus harder to read (bad!)